docs/source/whats-new.md

```diff
@@ -4,7 +4,93 @@ What's new
 See [GitHub releases page](https://github.com/OSOceanAcoustics/echopype/releases) for the complete history.
 
 
-# v0.8.3 (2024 December 24)
+# v0.9.0 (2024/07/20)
+
+## Overview
+This release includes a number of new features and performance enhancements, as well as fixes for bugs and warnings. We have fully switched to using the [trunk-based](https://www.atlassian.com/continuous-delivery/continuous-integration/trunk-based-development) development workflow in this release.
+
+## New features
+* Add noise and attenuation functions from Ryan et. al. 2015 and echopy by @ctuguinay in https://github.com/OSOceanAcoustics/echopype/pull/1316
+* Add `.bot` and `.idx` files to EK60/EK80 parsing by @ctuguinay in https://github.com/OSOceanAcoustics/echopype/pull/1318
+* Add index aggregation option for cleaning mask functions by @ctuguinay in https://github.com/OSOceanAcoustics/echopype/pull/1326
+* Added alpha-version code for new AZFP6 format by @dash-uvic in https://github.com/OSOceanAcoustics/echopype/pull/1323
+* Use latest MRU parser version and enable `ep.consolidate.add_location` can be based off datagram type by @ctuguinay in https://github.com/OSOceanAcoustics/echopype/pull/1359
+
+## Enhancements
+* Support lazy-loaded EK80 broadband-complex data by @leewujung in https://github.com/OSOceanAcoustics/echopype/pull/1311
+* Allow `MVBS` into `ep.mask.apply_mask`, and add alignment check for target variable and mask by @ctuguinay in https://github.com/OSOceanAcoustics/echopype/pull/1345
+* Add scalability options (`use_swap` and chunks) for `ep.calibrate.compute_Sv` by @ctuguinay in https://github.com/OSOceanAcoustics/echopype/pull/1331
```

Comment:
```suggestion
* Enchance ep.calibrate.compute_Sv scalability by enabling EchoData chunking and removing eager computation by @ctuguinay in https://github.com/OSOceanAcoustics/echopype/pull/1331
```

Reply:
I renamed this PR so that it reflects what the PR actually does. My initial naming of the PR was based on an earlier iteration of the code that had the swap functionality within the `compute_Sv` function.